### PR TITLE
Windows

### PR DESCRIPTION
**Pull request by eargollo**
- Adds support for Windows when using Windows Git bash
- Translates relative path on Windows to the Linux SSH host
- This PR also solves #74 as far as proxy is set at `.profile`

### CODE REVIEW

File: sshcode.go [hunk 1]
```diff
@@ -56,9 +56,10 @@ func sshCode(host, dir string, o options) error {
 	dlScript := downloadScript(codeServerPath)
 
 	// Downloads the latest code-server and allows it to be executed.
-	sshCmdStr := fmt.Sprintf("ssh %v %v '/usr/bin/env bash'", o.sshFlags, host)
+	// sshCmdStr := fmt.Sprintf("ssh %v %v /bin/bash -l", o.sshFlags, host)
+	sshCmdStr := fmt.Sprintf("ssh %v %v '/usr/bin/env bash -l'", o.sshFlags, host)
```
Comment (deansheather) [inline sshcode.go]: The login shell stuff should be in a separate PR as it's unrelated to Git Bash for Windows support.

File: sshcode.go [hunk 2]
```diff
@@ -56,9 +56,10 @@ func sshCode(host, dir string, o options) error {
...
 
-	sshCmd := exec.Command("sh", "-c", sshCmdStr)
+	sshCmd := exec.Command("sh", "-l", "-c", sshCmdStr)
```
Comment (deansheather) [inline sshcode.go]: SSH is called from more places than just this, so if we're going to change `sh` to a login shell so it reads the profile we should also do this every else.

**Comment by eargollo** [comment]
Closing this one to submit two separate PR's. Thanks for comments.
